mtp_noms_ops/apps/security/forms/check.py

```diff
@@ -74,17 +75,38 @@ def get_object_list(self):
         return object_list
 
 
-class ActionCheckForm(GARequestErrorReportingMixin, forms.Form):
+class AcceptOrRejectCheckForm(GARequestErrorReportingMixin, forms.Form):
     """
     Base CheckForm for accepting or rejecting a check.
     """
-    non_pending_error_msg = NotImplementedError
+    URGENT_IF_OLDER_THAN = datetime.timedelta(days=3)
+    decision_reason = forms.CharField(
+        label=gettext_lazy('Give details (details are optional when accepting)'),
+        required=False,
+    )
+    fiu_action = forms.CharField(max_length=10)
+
+    non_pending_error_msg = "You cannot action this credit as it's not in pending"
```

Comment:
this is now only used in one place, right? we might as well put the text there. also, looks like we had forgotten to wrap it in a `gettext` thing in the past

Reply:
The `gettext_lazy` was called where the variable was used so that's OK, but yeah we can just use the text in place of the variable now 👍